orchagent/portsorch.cpp

```diff
@@ -5609,9 +5609,13 @@ void PortsOrch::doTask(NotificationConsumer &consumer)
                 if (getPortOperSpeed(port, speed))
                 {
                     SWSS_LOG_NOTICE("%s oper speed is %d", port.m_alias.c_str(), speed);
-                    updateDbPortOperSpeed(port, speed);
+                    updatePortOperSpeedToStateDB(port, speed);
```

Comment:
I think we should keep the API as is and update to both APP and STATE DB

Reply:
Per our discussion, we will put the speed to STATE DB currently. I will rename the API back to previous.